manual/de/Help_ch_Account-Actions.xml

```diff
@@ -1369,25 +1369,6 @@
         </varlistentry>
       </variablelist>
     </sect2>
-
-    <sect2 id="accts-online-quotes">
-<!-- FIXME: move this section to section "tool-security-editor" -->
-      <title>Verwenden der Online-Kursaktualisierung</title>
-
-      <orderedlist>
-        <listitem>
-          <para>&app; bedient sich eines externen Tools zum Abrufen von Online-Kursen. Dieses Tool ist ein
-            &app-perl;-Modul namens &app-fq; und muss zusätzlich zu &app; auf Ihrem Computer
-            installiert sein.
-          </para>
-
-          <tip>
-            <para>Die Installation wird in <xref linkend="finance-quote" /> detailiert beschrieben.
-            </para>
-          </tip>
```

Comment:
Den Absatz würde ich aber bei den Wertpapieren wieder einfügen.

Reply:
Ist in der Beschreibung des Dialogs https://code.gnucash.org/docs/de/gnucash-help/tool-security-edit.html#tool-ge-GetOnlineQuotes doch enthalten.
Deshalb konnte ich jetzt diese Überbleibsel entfernen.

Reply:
Falsche Brille!

Da fallen mir aber 2 andere Sachen auf.

1. 2*`<note>` sieht unschön aus. Besser `<note><OrderedList>…`
2. Da schreibst du, man **muß** die Zeitzone setzen.

Reply:
Stimmt, ich arbeite noch deine Kommentare zu PR #260 durch. Kommt dann in dem neuen commit.